### PR TITLE
Add more logging to test deduplication

### PR DESCRIPTION
**Pull request by guynir42**

### CODE REVIEW

**Review by mcoughlin — APPROVED** [review]
LGTM

File: skyportal/handlers/api/photometry.py [hunk 1]
```diff
@@ -969,9 +972,16 @@ def put(self):
                                         photometr_id=duplicate.id, stream_id=id
                                     )
                                 )
+                            log(
+                                f'Adding streams {stream_ids_update} to photometry {duplicate.id}'
+                            )
 
                 # now safely drop the duplicates:
                 new_photometry = df.loc[new_photometry_df_idxs]
+                print(
```
Comment (mcoughlin) [inline skyportal/handlers/api/photometry.py:981]: Should this be a log maybe @guynir42 ?

Comment (guynir42) [reply skyportal/handlers/api/photometry.py:981]: yeah that's right!

**Comment by guynir42** [comment]
@mcoughlin can we close this PR as it is already doing the same thing in yours?

**Comment by mcoughlin** [comment]
@guynir42 You added a few more lines right?

**Comment by guynir42** [comment]
@mcoughlin yes but I assumed you started this PR on top of mine. Nevermind I will resolve the conflicts then merge this in.

**Pull request merged**